api/database/migrations/2023_01_27_201348_laratrust_setup_teams.php

```diff
@@ -0,0 +1,76 @@
+<?php
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+
+class LaratrustSetupTeams extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        // Create table for storing teams
+        Schema::create('teams', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('name')->unique();
+            $table->string('display_name')->nullable();
```

Comment:
I think it would be best to update this to support our `LocalizedString` type.

Reply:
hm, okay
I made changes to the migrations in https://github.com/GCTC-NTGC/gc-digital-talent/pull/5527/commits/2c5ef76f71a2dba2e92e483b17a1e78c722c12ce

Reply:
going off the desired type of Team in https://github.com/gctc-ntgc/gc-digital-talent/issues/5511
I made the change for display_name and description